tools/ci_build/github/linux/run_build.sh

```diff
@@ -24,10 +25,17 @@ if [ $BUILD_DEVICE = "gpu" ]; then
         --cudnn_home /usr/local/cudnn-$_CUDNN_VERSION/cuda --build_shared_lib $BUILD_EXTR_PAR
     /home/onnxruntimedev/Release/onnx_test_runner -e cuda /data/onnx
 else
-    python3 $SCRIPT_DIR/../../build.py --build_dir /home/onnxruntimedev \
-        --config Debug Release --build_shared_lib \
-        --skip_submodule_sync --enable_onnx_tests \
-        --enable_pybind \
-        --parallel --use_openmp --build_shared_lib $BUILD_EXTR_PAR
-    /home/onnxruntimedev/Release/onnx_test_runner /data/onnx
+    if [ MACHINE = "x86_64" ]; then
```

Comment:
The only difference between the if/else clause is the omission of enable_pybind and enable_onnx_tests.

Reply:
--enable_onnx_tests is the only difference.
Failing cases are test_vgg19_cpu, test_zfnet512_cpu.
In the meantime, some tests give warning like - "ONNX Runtime supports 64bit architecture"

Reply:
https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=9922

Reply:
In the Windows case, I manually ran the failed test cases on my machine and they passed, but we disabled them in CI temporarily. Are these tests passing on your local machine? Can you verify? You can run one test like as follows:

``` .\build\Windows\RelWithDebInfo\RelWithDebInfo\onnx_test_runner -j 1 -n <testname> <onnx model folder> C:\Users\prs\work_projects\onnxruntime\cmake\external\onnx\onnx\backend\test\data```